script/core.hbs

```diff
@@ -28,10 +30,32 @@
         {{/if}}
       {{/with}}
     };
+    {{#if global_config.useJWT}}
+      const jwtNotProvidedTimeout = setTimeout(() => {
+        console.error(
+          'Since useJWT is set, a JWT is required to initialize this Answers experience.\n' +
+          'To specify a JWT, this page must be loaded through iframe-jwt.js or similar.'
+        );
+      }, 2000);
```

Comment:
Should we use a larger timeout here, or is it very unlikely to get a false positive here even on slow connections?

Reply:
It's quite a safe timeout. I didn't get any false positives even when setting the network to 'Slow 3G' and the cpu to '6x slowdown'. The timer starts ticking when the experience loads, and it stops once the iframe receives the JWT from the parent. So the timeout only has to be larger than the time it takes for the iframeResizer to send the initial message from the parent to the iframe, which is quick.

Reply:
Since this PR now supports non-iframe experiences, I've bumped the timeout to 5 seconds and am switching it to a warning rather than an error.